app/assets/javascripts/components/registration_modal.es6.jsx

```diff
@@ -171,20 +171,20 @@ class RegistrationModal extends React.Component {
     }
     return (
       <section className="signup">
-        <div className="container signup-container">
+        <div className="signup-container">
           <div className="signup-row">
           <h3>Create a Profile</h3>
             <form>
               <label className="required-field">* Required</label>
               <div className="input-field">{ this._renderInput("first_name", "First Name", "text", "John", required=true) }</div>
               <div className="input-field">{ this._renderInput("last_name", "Last Name", "text", "Doe", required=true) }</div>
               <div className="input-field">{ this._renderInput("email", "Email", "text", "lavamae@gmail.com", required=true) }</div>
+              <div className="input-field">{ this._renderInput("password", "Password (min. 8 characters)", "password", "", required=true) }</div>
+              <div className="input-field">{ this._renderInput("password_confirmation", "Confirm Password", "password", "", required=true) }</div>
               <div className="input-field">{ this._renderInput("secondary_name", "Name of Secondary Contact", "text", "Jane Doe") }</div>
```

Comment:
move secondary and tertiary input fields to end (after location, but before checkboxes)